lib/utils/to.js

```diff
@@ -71,10 +77,11 @@ module.exports = {
       return val;
     }
     if (typeof val == "string") {
-      if (val.indexOf('0x') != 0) {
-        return parseInt(val)
+      if (val.indexOf('0x') !== 0) {
+        return parseInt(val, 10)
```

Comment:
Is it possible that val exceed's Number.MAX_SAFE_INTEGER here?

Reply:
Yes. May we RIP if and when this happens. I'll hopefully handle these edge cases in the upcoming refactor.